test/users.test.js

```diff
@@ -45,14 +47,15 @@ describe('POST /users', () => {
             firstName: 'Jose',
             lastName: 'Perez',
             email: 'jose.perez@wolox.com.ar',
-            password: '12345test'
+            password: '12345test',
+            confirm_password: '12345test'
           })
           .then(response => {
-            const { message, internal_code } = response.body;
-            expect({ status: response.statusCode, message, internal_code }).toStrictEqual({
-              status: 503,
-              message: 'Error processing request in database.',
-              internal_code: 'database_error'
+            const { message, internalCode } = response.body;
+            expect({ status: response.statusCode, message, internalCode }).toStrictEqual({
+              status: 400,
+              message: 'Invalid request parameters',
```

Comment:
This is not the same with what we talked about. You must be send an array for errors in message :( . Check this in all test